.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,6 +2,7 @@
 
 1. [ ] Code follows the coding style of [CONTRIBUTING.md][contributing]
 1. [ ] Tests follow the [best practice for testing][testing]
+1. [ ] Each unit test case has a UUID
```

Comment:
It would be nice if this works but I bet this one will be checked without thinking. We either need the CI to check this or a task for the reviewer :/